modules/appeals_api/app/models/appeals_api/non_veteran_claimant.rb

```diff
@@ -48,21 +48,33 @@ def number_and_street
     end
 
     def city
-      form_data['address']['city']
+      form_data.dig('address', 'city')
     end
 
     def state_code
-      form_data['address']['stateCode']
+      form_data.dig('address', 'stateCode')
     end
 
     def country_code
+      return if form_data.blank?
```

Comment:
You should be checking that the specific data you're looking for (e.g. `form_data['address']['countryCodeISO2']`) is missing, not against all the form data being blank.

Reply:
I don't want it to print 'US' though, if there is no form data.

Reply:
i'll see if there's a nicer way to do this.

Reply:
As it stands now, you'd still be printing `US` even if an unrelated form_data key existed, like `form_data['email']`. You only want to return nil if the specific data you're trying to otherwise print is not present.

Reply:
RIGHT!